modules/govuk_jenkins/templates/jobs/deploy_puppet_downstream.yaml.erb

```diff
@@ -0,0 +1,35 @@
+---
+- job:
+    name: Deploy_Puppet_Downstream
+    project-type: freestyle
+    description: "Kicks off a Puppet deploy in a downstream environment"
```

Comment:
I'm finding the name and description quite confusing - am I right in thinking this job is to trigger the "Deploy puppet" jenkins job on the "downstream" jenkins? From the current name, I'd think that is job is actually doing the work to "deploy puppet". 

Can we be more specific in the name and description? e.g. Trigger_Deploy_Puppet_Downstream

Reply:
Hmm...this would be quite time consuming to change, as we should make it consistent with the existing `Deploy_App_Downstream` job. I think it's helpful that they both begin with `Deploy_X`, so that these related jobs are grouped together in the UI.

If we do change the name, I'd like to stick with a `Deploy_App` prefix. What about `Deploy_Puppet_Downstream_Trigger`?